src/crimson/os/seastore/transaction_manager.h

```diff
@@ -635,6 +544,27 @@ class TransactionManager : public ExtentCallbackInterface {
     });
   }
 
+  reserve_extent_ret reserve_region(
+    Transaction &t,
+    LBAMapping mapping,
```

Comment:
For inserts, mapping -> next_mapping

let's make the name consistent in other methods as well.